src/Features/ERDDAP/Platform/Forecasts/index.tsx

```diff
@@ -1 +1,172 @@
-export { Forecast } from "./Page"
+/**
+ * Load and display forecasts
+ */
+import { Point } from "@turf/helpers"
+import * as React from "react"
+import { Alert, Row, Col } from "reactstrap"
+
+import { MultipleLargeTimeSeriesChart } from "components/Charts"
+import { round } from "Shared/math"
+import { aDayAgoRounded } from "Shared/time"
+import { DataTimeSeries } from "Shared/timeSeries"
+import { UnitSystem } from "Features/Units/types"
+import { converter } from "Features/Units/Converter"
+
+import { useDataset, useForecast, useForecasts } from "../../hooks"
+import { ForecastSource, PlatformFeature, PlatformTimeSeries } from "../../types"
+
+interface Props {
+  platform: PlatformFeature
+  forecast_type: string
+  unitSystem: UnitSystem
+}
+
+/**
+ * Match relevant forecast to platform datasets
+ */
+export const Forecast: React.FunctionComponent<Props> = ({ platform, forecast_type, ...props }) => {
```

Comment:
Codacy found an issue: [Identifier 'forecast_type' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6288046)